src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -141,7 +143,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	private readonly _processManager: ITerminalProcessManager;
 	private readonly _resource: URI;
 	private _shutdownPersistentProcessId: number | undefined;
-
+	get processManager() { return this._processManager; }
```

Comment:
We don't want to expose this, is it just used in tests? If so _processManager could be protected and exposed in a sub class.

Reply:
hm tried that and not sure how to get that to work, so exposed the function on terminal instance